definition_of_done.md

```diff
@@ -0,0 +1,37 @@
+# Definition of done
+
+This document clarifies, when a product backlog item is done. It will act as a checklist during the scrum development process.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api function descriptions (sphinx/doxygen are [autogenerated online](http://docs.ros.org/melodic/api/pilz_robots/html/))
+* [ ] API diagrams
+* [ ] [Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)
+* [ ] Overview on [ROS wiki](https://wiki.ros.org/pilz_robots)
+* [ ] Package Readme ([example pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md))
+* [ ] Good commit messages ([some tips](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9
+))
+* [ ] Changelog.md updated
+* [ ] Copyright headers
+
+## Reviews
+* [ ] Soft- and hardware architecture
+* [ ] Code (coding rules, style guide)
+* [ ] Test review (test plan and individual test cases)
+* [ ] Documentation review
+
+## Quality
+* [ ] CI pass:
+The travis job (for example in [pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/.travis.yml)) checks full code coverage, successful test execution and code formatting.
+
+## Demo
+* [ ] Example program / snippet prepared ([example at pilz_robot_programming](http://docs.ros.org/melodic/api/pilz_robot_programming/html/))
+* [ ] Ready for sprint review
+
+## Release planning
+* [ ] When is the new feature released?
+* [ ] Which dependent packages do have to be released simultaneously?
+
+## Cleanup
+* [ ] [Squash-Merge](https://github.blog/2016-04-01-squash-your-commits/)
```

Comment:
I would like to emphasize the use of the `squash-merge` feature in github. Rather than squashing the feature branch manually. this keeps the edit history in the feature branch, which is deleted, anyway but could be restored if needed later.